tools-local/tasks/BuildFPMToolPreReqs.Netstandard2_0.cs

```diff
@@ -0,0 +1,581 @@
+                }
+            }
+
+            return configJson;
+        }
+
+        internal static void ReadStartObject(ref Utf8JsonReader reader)
+        {
+            reader.Read();
+            CheckStartObject(ref reader);
+        }
+
+        internal static void CheckStartObject(ref Utf8JsonReader reader)
+        {
+            if (reader.TokenType != JsonTokenType.StartObject)
+            {
+                throw CreateUnexpectedException(ref reader, "{");
+            }
+        }
+
+        internal static void ReadStartArray(ref Utf8JsonReader reader)
+        {
+            reader.Read();
+            CheckStartArray(ref reader);
+        }
+
+        internal static void CheckStartArray(ref Utf8JsonReader reader)
+        {
+            if (reader.TokenType != JsonTokenType.StartObject)
+            {
+                throw CreateUnexpectedException(ref reader, "[");
+            }
+        }
+
+        internal static string ReadAsString(ref Utf8JsonReader reader)
+        {
+            reader.Read();
+            if (reader.TokenType != JsonTokenType.String)
+            {
+                throw CreateUnexpectedException(ref reader, "a JSON string token");
+            }
+            return reader.GetStringValue();
+        }
+
+        internal static Exception CreateUnexpectedException(ref Utf8JsonReader reader, string expected)
+        {
+            return new FormatException($"Unexpected character encountered, excepted '{expected}' " +
+                                       $"at line {reader.CurrentState._lineNumber} position {reader.CurrentState._bytePositionInLine}");
+        }
+
+        // Update the tokens in the changelog file from the config Json 
+        private void UpdateChangelog(ConfigJson configJson, string package_version)
+        {
+            try
+            {
+                string changelogFile = Path.Combine(InputDir, "templates", "changelog");
+                if (!File.Exists(changelogFile))
+                {
+                    throw new FileNotFoundException($"Expected file {changelogFile} was not found.");
+                }
+                string str = File.ReadAllText(changelogFile);
+                str = str.Replace("{PACKAGE_NAME}", configJson.Package_Name);
+                str = str.Replace("{PACKAGE_VERSION}", package_version);
+                str = str.Replace("{PACKAGE_REVISION}", configJson.Release.Package_Revision);
+                str = str.Replace("{URGENCY}", configJson.Release.Urgency);
+                str = str.Replace("{CHANGELOG_MESSAGE}", configJson.Release.Changelog_Message);
+                str = str.Replace("{MAINTAINER_NAME}", configJson.Maintainer_Name);
+                str = str.Replace("{MAINTAINER_EMAIL}", configJson.Maintainer_Email);
+                // The date format needs to be like Wed May 17 2017
+                str = str.Replace("{DATE}", DateTime.UtcNow.ToString("ddd MMM dd yyyy"));
```

Comment:
Ok, we obviously don't care about perf in this code :)

Reply:
Yep, doesn't look like it from what I gather.